style.css

```diff
@@ -0,0 +1,695 @@
+    border: purple 1px solid;
+    border-radius: 50px;
+    font-size: 2em;
+    margin: 1.2rem;
+}
+
+.header-contact a:hover,
+.header-contact a:active {
+    background: 0;
+    color: purple;
+    border: purple 1px solid;
+    transition: ease-in-out .5s all;
+}
+
+/*-------------------------------------------------*/
+/* section about me & skills */
+/*-------------------------------------------------*/
+
+.section-about {
+    display: flex;
+    flex-direction: column;
+}
+
+/* about me */
+.section-about-me {
+    background-color: #fafaff;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 92%;
+    height: 50%;
+    margin-top: 2%;
+}
+
+.section-about > p {
+    font-weight: 700;
+    font-size: 200%;
+}
+
+.section-about p {
+    display: inline-block;
+    padding-left: 1%;
+    color: #ffffff;
+    margin-top: 1em;
+    color: purple;
+}
+
+.section-about-me-left {
+    padding: 10px 0;
+    margin: 0;
+}
+
+/* about me let side */
+.section-about-me-left {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.section-about-me-left > img {
+    width: 12em;
+    height: 12em;
+    margin: 1em;
+    border-radius: 50%;
+    display: flex;
+    align-self: center;
+}
+
+.section-about-me-left-ul {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    align-items: center;
+    width: 50%;
+    margin: auto 0;
+}
+
+.section-about-me-left-ul li {
+    list-style: none;
+    font-size: 1.5em;
+    color: purple;
+    padding: .8rem;
+}
+
+.section-about-me-left-ul li span {
+    color: rgb(36, 36, 36);
+    font-weight: 700;
+}
+
+/* about me right */
+.section-about-me-right {
+    padding: 0 1rem 1rem 1rem;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.04em;
+    width: 50%;
+}
+
+/* text about me (right side) */
+.section-about-me-right p {
+    font-size: 1.2em;
+    padding: 1em;
+    text-align: justify;
+    border-left: .4rem dotted purple;
+}
+
+/* skills*/
+.section-skills {
+    float: right;
+    background-color: #fafaff;
+    padding: 3em;
+    margin-top: 2%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: baseline;
+    width: 85%;
+    align-self: flex-end;
+    color: purple;
+    height: 50%;
+}
+
+.section-skills-left {
+    border-right: .4rem dotted rgb(143, 77, 143);
+    padding: 1em;
+    text-align: center;
+    width: 32%;
+    font-size: 1.5rem;
+    font-weight: 700;
+}
+
+.section-skills-center {
+    border-right: .4rem dotted rgb(143, 77, 143);
+    padding: 1em;
+    text-align: center;
+    width: 32%;
+    font-size: 1.5rem;
+    font-weight: 700;
+}
+
+.section-skills-right {
+    width: 32%;
+    padding: 1em;
+    text-align: center;
+    font-size: 1.5rem;
+    font-weight: 700;
+}
+
+.section-skills img {
+    height: 100px;
+}
+
+/*-------------------------------------------------*/
```

Comment:
Jak wprowadziłeś komentarze, to znaczy, że ten CSS jest za duży. Mógłbyś ewentualnie spróbować podzielić go na więcej.